File: fixtures/scala/baseline/src/main/scala/example/Service.scala
package example

object Defaults {
  val Prefix = "job"
}

class Repository {
  var last: String = ""

  def save(value: String): String = {
    last = value.trim
    last
  }
}

class Service(repository: Repository) {
  def execute(name: String): String = {
    val stored = repository.save(name)
    s"${Defaults.Prefix}:$stored"
  }
}

object Service {
  def build(repository: Repository): Service = new Service(repository)
}

File: fixtures/python/baseline/src/example/service.py
DEFAULT_PREFIX = "job"

class Repository:
    def __init__(self):
        self.last = ""

    def save(self, value):
        self.last = value
        return value.strip()

class Service:
    def __init__(self, repository):
        self.repository = repository

    def execute(self, name):
        stored = self.repository.save(name)
        return f"{DEFAULT_PREFIX}:{stored}"

def build_service():
    return Service(Repository())
